contracts/protocol/libraries/logic/ValidationLogic.sol

```diff
@@ -252,6 +252,17 @@ library ValidationLogic {
             Errors.PRICE_ORACLE_SENTINEL_CHECK_FAILED
         );
 
+        if (params.releaseUnderlying) {
+            vars.availableLiquidity = IERC20(params.asset).balanceOf(
+                params.reserveCache.xTokenAddress
+            );
+
+            require(
```

Comment:
if we already check collateral can cover debt [here](https://github.com/para-space/paraspace-core/blob/da9d57291c01f33de106819347a624f4ae84b181/contracts/protocol/libraries/logic/ValidationLogic.sol#L327-L331) do we still need this?